libs.autobuild

```diff
@@ -39,6 +39,7 @@ in_flavor 'master','stable' do
 
     mars_package("simulation/mars/scripts/cmake")
     mars_package("simulation/mars/common/cfg_manager")
+    mars_package("simulation/mars/common/gui/config_map_gui")
```

Comment:
I assume it is not (Yet) in the stable branch, therefore this would cause compile problems oon master, please make sure you put it in the right section and rebase this PR